src/components/ManageUsersView.tsx

```diff
@@ -28,8 +28,9 @@ function ManageUsersView({ classes, removeUser, users }: ManageUsersProps) {
           color="primary"
           label={user}
           key={user}
-          onDelete={removeUser}
+          onDelete={() => (removeUser ? removeUser(user) : null)}
```

Comment:
If desired, this could be simplified a little using [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining).

```suggestion
          onDelete={() => removeUser?.(user)}
```

Reply:
Oh, nice! That is much cleaner, thank you!

Reply:
I didn't know you could do this on functions. Is this syntax typically used on functions (e.g. in the Terra UI codebase)?

Reply:
`git grep '?\.(' | wc -l` shows it's used 20 times in Terra UI.